src/changelog.ts

```diff
@@ -98,17 +98,17 @@ export default class Changelog {
     return Git.listCommits(from, to);
   }
 
-  private async getCommitters(commits: CommitInfo[]): Promise<GitHubUserResponse[]> {
```

Comment:
The name of the interface (`GitHubUserResponse`, `GitHubIssueResponse`) was also abstracted and changed. (`GitHostingUserResponse`, `GitHostingIssueResponse`)